### DISCUSSION TITLE
Error Installing Example Template

### DISCUSSION THREAD

**Discussion by ElCidJJ** [Syslifters/sysreptor#255] [General]
Good Afternoon,

I would like to install the example HTB design on the [website](https://docs.sysreptor.com/htb-reporting-with-sysreptor/) according to these instructions:

`cd sysreptor/deploy`
`url="https://docs.sysreptor.com/assets/htb-designs.tar.gz"`
`curl -s "$url" | docker compose exec --no-TTY app python3 manage.py importdemodata --type=design`

When I do this, it shows me the following error message:
`WARN[0000] /DEPLOY/CONFIG/sysreptor/sysreptor/deploy/docker-compose.yml: `version` is obsolete `
`WARN[0000] /DEPLOY/CONFIG/sysreptor/sysreptor/deploy/docker-compose.override.yml: `version` is obsolete`

I see that I have everything updated to the latest version, in my case, I have the docker-compose.yml in version 3.9.

How could I solve this problem?

Thanks

**Comment by MWedl** [comment]
Hi,
you can ignore this warning. The designs should be imported anyway.

Docker-compose deprected the `version` entry in `docker-compose.yml` files. We removed it from our docker-compose files in the latest version. If you update SysReptor, the warning should disappear. See https://docs.sysreptor.com/setup/updates/

**Comment by ElCidJJ** [comment]
Thanks for the reply.

Now it happens to me that after trying it several times, each design has been installed 3 times. Is there a way to delete the designs?

**Reply by MWedl** [reply]
You can delete them via the web interface. The delete button is in the 3-dot-menu on the design settings page.

![image](https://github.com/Syslifters/sysreptor/assets/29510012/d2e85e07-57e4-4bdc-b514-52588d0d2f60)

**Comment by ElCidJJ** [comment]
Thank you so much.

I'm new to this and I'm having a hard time getting started.
